tests/test_mig_install_generateconfs.py

```diff
@@ -0,0 +1,44 @@
+from __future__ import print_function
```

Comment:
Missing standard copyright header. I'll add one this time.

Reply:
Damn it - my bad and I hold my hands up. Will not mark things for review without that going forward.